library/varnish

```diff
@@ -1,26 +1,26 @@
-# this file was generated using https://github.com/varnish/docker-varnish/blob/d1212e4b8fd35b58c19b01ed389f8841d0a4ea38/populate.sh
-Maintainers: Guillaume Quintard <guillaume@varni.sh> (@gquintard)
+# this file was generated using https://github.com/varnish/docker-varnish/blob/693cddb1140d5c0e042793cbb92b6906fb8425a5/populate.sh
+Maintainers: Guillaume Quintard <guillaume.quintard@gmail.com> (@gquintard)
 GitRepo: https://github.com/varnish/docker-varnish.git
 
-Tags: fresh, 7.0.2, 7.0, latest
+Tags: fresh, 7.1.0, 7.1, latest
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: fresh/debian
-GitCommit: 6423e32e0afd20fa876276e1525cb318920fbefa
+GitCommit: bb05db3ed5d75a1d8402309336823e2d58330b1b
 
-Tags: fresh-alpine, 7.0.2-alpine, 7.0-alpine, alpine
+Tags: fresh-alpine, 7.1.0-alpine, 7.1-alpine, alpine
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: fresh/alpine
-GitCommit: 6423e32e0afd20fa876276e1525cb318920fbefa
+GitCommit: 693cddb1140d5c0e042793cbb92b6906fb8425a5
```

Comment:
I think you meant to go a little bit higher here? (to pick up that `alpine:3.15` change in https://github.com/varnish/docker-varnish/commit/2ac414bf99af92cabbf4a924108f0c3af190f47f)

```suggestion
GitCommit: 2ac414bf99af92cabbf4a924108f0c3af190f47f
```